docs/mindsdb_sql/agents/agent_syntax.mdx

```diff
@@ -80,6 +80,18 @@ Available models:
 - claude-instant-1.2
 </Accordion>
 
+<Accordion title="Bedrock">
+Available models include all models accessible from Bedrock.
+
+Note that in order to use Bedrock as a model provider, you should ensure the following packages are installed: `langchain_aws` and `transformers`.
+
+The following parameters are specific to this provider:
+
+* `aws_region_name`
+* `aws_access_key_id`
+* `aws_secret_access_key`
+</Accordion>
```

Comment:
there is one more parameter available:
- aws_session_token
it depends on how aws premissions are setup, sometimes it required, somethimes - not
for example was required [here](https://github.com/mindsdb/mindsdb/pull/10855)

Reply:
added it in